charts/rancher-gatekeeper/100.0.0+up3.5.1/Chart.yaml

```diff
@@ -2,10 +2,11 @@ annotations:
   catalog.cattle.io/auto-install: rancher-gatekeeper-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: OPA Gatekeeper
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.22.0-0'
   catalog.cattle.io/namespace: cattle-gatekeeper-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: config.gatekeeper.sh.config/v1alpha1
-  catalog.cattle.io/rancher-version: < 2.6.1-0
```

Comment:
This seems like an intentional change to hide the chart after 2.6.1 right? We should look into why this was necessary, but based on this we should be releasing:

```
'>= 2.6.0-0 < 2.6.1-0'
```

Reply:
Good catch, definitely missed that < in this one!